assets/styles/components/specials/_textboxes.scss

```diff
@@ -96,6 +96,7 @@
     line-height: if-map-get($textbox-line-height, $type);
   }
   text-align: $textbox-text-align;
+  text-indent: if-map-get($textbox-text-indent, $type);
```

Comment:
Probably should be on the `.textbox p` selector, not the `.textbox` selector. And should support the indend/skiplines paragraph separation theme options.